prime-router/settings/organizations.yml

```diff
@@ -2418,6 +2418,11 @@
   filters:
     - topic: covid-19
       jurisdictionalFilter: [ "matches(ordering_facility_state, IG)" ]
+    - topic: full-elr
+      jurisdictionalFilter:
+        - "%performerState = 'IG' or %patientState = 'IG'"
+      qualityFilter:
+        - "%hl7MessageType = 'R01'"
```

Comment:
This is fine as a test here, but just FYI that this is not a proper quality filter for a receiver.  Just think that the system is ingesting all sorts of data and this receiver is getting information that a bunch of data was filtered just because it was not an observation.